### DISCUSSION TITLE
Floodplain Condition O:E

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/QRiS#212] [Specification Discussion]
@chris3jordan asked @emilyfairfax and I in an email these important questions:
> I keep poking in my own head at the idea of some metric of riverscape condition that is of the Observed/Expected form. Mainly I want to be able to reinforce the assertion that most CONUS floodplains are impaired, primarily by simplification / starvation, but regardless of
pathway, less than ideal. 
>
> VBET current / historic.
>
> channel length per unit valley length current / historic
>
> later summer floodplain NDVI current / historic
>
> Aggregate process metrics that indicate lateral and vertical connectivity.
>
>Thoughts?

Does @joewheaton have thoughts? Silly question... you'll regret asking. I posted this here (in QRiS discussion) for two reasons. First, because this is where we are most likely to facilitate being able to answer this question and do this analysis at the project scale. 

Secondly, so we can get more of the team weighing in. Specifically this is something that @philipbaileynar and @KellyMWhitehead are in the process of thinking deeply and specifically about in QRiS development. @nick4rivers, @coloradomurph, @stephen-bennett, @nbouwes and @Jdgilby are all working on versions of this question in various monitoring efforts Anabranch is involved with.  In the @Riverscapes/etal-fhc, @Cashe93, @wally-mac, @leallysmith, @nbouwes, Hayley Glassic and Robert Al-Chokachy are all working on as part of the LCT project in Nevada.  @AldenShallcross and Scott Miller and I are also scrambling on this one.  So I'm hoping some of those voices weigh in here.

**Comment by joewheaton** [comment]
Thanks CJ for this. I don't know if I like this as a "floodplain condition" question because the size of the floodplain (its sample frame) is always fluctuating. My first advice is to reframe this as a "riverscape condition", within which a metric getting at riverscape condition that looks at the relative extent of the floodplain is considered.

> > I keep poking in my own head at the idea of some metric of riverscape condition that is of the Observed/Expected form. Mainly I want to be able to reinforce the assertion that most CONUS floodplains are impaired, primarily by simplification / starvation, but regardless of pathway, less than ideal.

So, I think any question of observed : expected is way more tractable at the local "my riverscape" reach-scale where we can reasonably come up with some historical estimate of "**E**" for comparison for the metric we're interested (the "**O**" is so easy to measure directly we don't even need to go there).  This is mainly why I roped this discussion in here to QRiS, because I think we stand a better chance of cracking this nut sooner and getting this as a tool in Riverscapes Studio. 

## Riverscapes Network Scale Approaches
I hear you on CONUS and wanting to check if you assertion is reasonable.  Within the next few weeks we should finally have the primary  waterfall of [Production Grade Riverscape Network Models](https://tools.riverscapes.net) run for 2/3 of the US (i.e. Mississippi Basin and everything West; so hoping you don't care about Eastern Seaboard). 

![Riverscapes Models Waterfall](https://user-images.githubusercontent.com/9723905/231475877-d12fa8c0-e641-4e10-92ba-41a67b3f0586.png)

![CONUS_ModelCoverage](https://user-images.githubusercontent.com/9723905/231476777-3e21c10e-0d4e-4f81-badf-b67def904d5d.png)


Now, out of VBET 3.x, we get one measure (a riverscape-wide measure) which I would argue is our most defensible and tractable framing of upper limits on **E** at broad spatial extents. That is we get the valley bottom. The maximum "active" floodplain extent is obviously bounded by that. The problem, as the LCT folks will be quick to point out, is that in really arid areas with smaller hydrologic engines naturally,  you would not have historically expected the entire valley bottom to be active, but it all would have been accessible. It is a start.

Also now, VBET will give you sample frames (in Discrete Geographic Objects - DGOs and moving windows as multiples of DGOs e.g. 3x DGO, 5xDGO, 7xDGO etc., Integrated Geographic Object - IGOs) from which to calculate these metrics everywhere.  So now for every DGO and IGO, we have:
- Valley Bottom Width
- Valley Bottom Area
- Riverscape CL Length
- Channel Length (primary & total)
- A thresholded VBET relative likelihood into
  - proportion of VB in channel
  - proportion of VB in low-lying valley bottom (formerly sloppily called active)
  - proportion of VB in elevated valley bottom

We need to be careful here as we don't over interpret what we can get out of 10 m DEMs. In some places, this is an okay approximation of condition and proportion active. In a lot of places it is not. 

@jtgilbert and @shelbysawyer and @lauren-herbine and I have recently changed the terminology from "Active" vs. "Inactive" floodplain in VBET, because it is misleading. We will eventually get back to approximations of "active", but we like to say that VBET is just identifying the part of the floodplain that low-lying versus elevated. Why the distinction. We want to obscrure this from how we normally talk about active floodplains as being defined by some mix of how a) "easily" it can flood (partly hydrology, which we don't look at, and partly topography, which we do look at), b) ability to support riparian or wetland vegetation (we have an approximation of this in RS_Context from LANDFIRE EVT and use it in RCAT and BRAT), and the ability for the channel to adjust its position within the valley bottom (we have a proxy for this in floodplain accessibility now in the improved RCAT). 

So for Proportion Active (**O**), measuring it is straight forward (LCT crew has done a ton of this) and comparing or normalizing this by valley bottom width/area is really easy now in both VBET and QRiS.  But what you consider **E** is tricky. In wetter climates and some riverscapes it is likely reasonably to assume that the range for your **E** is up to 100%. But you at least know its not more then that. If you can assume it to be 100%, or just live with this and side step it as your "historic expectation" but just treat it as a "maximum expectation", you now have a way out for this one. 

We will eventually get back to a proxy for active (probably 2 years from now) and we will do that in @lauren-herbine baby → "Riverscapes Metric Engine", formerly known to some as GNAT. This is where will calculate both channel segment metrics as well as riverscape network metrics (from DGOs onto IGOs). It will ingest values from upstream models (e.g. VBET, RCAT, Anthro, BRAT). So we will likely . This is also where the confinement tool will be housed. 

## Specific Metrics you Suggested

> > VBET current / historic.

This doesn't make any sense @chris3jordan. VBET seeks to map the riverscape or valley bottom defined as the part of the landscape that could plausibly flood in the contemporary natural flow regime. 

> > channel length per unit valley length current / historic

We give you channel length per unit valley length currently no problem now in VBET if you're happy with the NHD+HR map as a proxy of existing. However, what basis do you have a broad extents for coming up with historic? I can't think of any. This was the fallacy of the whole non-sense that Imaki and Beechie claimed.  About the only approach that makes sense to me is to attempt to reach type and attempt to find modern day reference conditions that can be assumed to representative, and attempt to apply those everywhere. That is a shit ton of work (like a thesis) and a bit overkill for your problem. 

If you are comfortable tracing historic channels off of HRT or nice imagery, go for it. This is the perfect and easy thing to do in QRiS (FYI @KellyMWhitehead and @philipbaileynar).

> > later summer floodplain NDVI current / historic

Again @chris3Jrodan, NDVI current (back to 1984 when things were already screwed up) is no problem. Prior to that forget it. You have no basis for getting historic NDVI values to use as an **E**. The closest thing is what we do with Riparian Vegetation Departure (RVD) in RCAT. @AldenShallcross has paid for RCAT for Montana/Dakotas, but you don't have to wait for RCAT to exist everywhere to see this data. We moved it into the RS_Context and that is available now (e.g. [this project from West Virginia](https://staging.warehouse.riverscapes.net/p/2f0a35c5-25d3-4541-82f1-912c0ee7f9c1/))

![image](https://user-images.githubusercontent.com/9723905/231484185-29a09fd8-d515-49fb-92ac-eba04374efcb.png)

> > Aggregate process metrics that indicate lateral and vertical connectivity.

Yes, the one I mentioned to eventually get at "Proportion Active". We're going to dress this one up coming straight out of national NED 10m driven VBET as something more obscure sounding. In some cases it is a good approximation of proportion active, but in many it is not and this is better done with higher resolution data or down in the Studio (QRiS) at local reaches.

**Reply by chris3jordan** [reply]
Ok, granted, my language was sloppy, but to be fair, I wasn't starting a threaded discussion I was dashing off an email to @joewheaton and @emilyfairfax that was then converted into this more formal conversation. 

So, let me start again more simply and carefully.

We, the riverscapes community, would benefit from an index of impairment for riverscapes. 

The design criteria for this riverscape impairment index are {@everyoneelse add here}:

1. The impairment index should respond in a context independent manner, but clearly be context dependent.
2. The impairment index should index the degree of anthropogenic impairment of riverscape function broadly.
3. Alt #2 - it is not possible to generate a single, broadly representative (in the dimensions of impairment, not in the dimensions of context) index, therefore we should develop a family of related departure indices, each referencing an orthogonal axis of the impairment space. For example, using the Castro-Thorne stream evolution triangle as our source of basis functions, there would be three indices, one each for the fundamental dimensions, (biology, flow, geology).

The power of O/E lies in the simplicity of a non-dimensional index of departure. 

However, let's not conflate <historic> with <expected>.

Current <observed> versus something that we think it should be, whether this alternate state is old or ideal doesn't matter, and if we stay away from <historic>, then we avoid the petty discussions of "when was a riverscape <historic>".

What this index actually indexes clearly depends on our concept of what is relevant to describe a riverscape and its functional state.

For example, channel size or location is irrelevant, but access to water by floodplain vegetation at base flow is very relevant. Therefore, we don't need to measure or predict features of a channel, or features that generate or control channel geometry or location, but we do need to measure or predict riverscape features that generate or control the vertical and lateral distribution of down-valley water (surface or hyporheic flowing).

In this context, VBET_O/E and NDVI_O/E do make sense, even if we don't right now know how to represent <NDVI_expected> at the moment. We certainly know both parts of VBET_O/E, since the majority of departures on the <VBET_observed> side come from infrastructure modifications of the valley bottom surface. 

Some thoughts on what an <NDVI_expected> raster might be built from include regional climate driven expectations for lateral extent of down-valley water at low flow such that <NDVI_expected> deviates from maximal greenness/wetness due to low Q and high EVT. 

A parallel thought:

EPA has done considerable work on CONUS O/E indices based on their NARS program and StreamCat dataset. I think it would be appropriate to bring them into this conversation.

**Comment by leallysmith** [comment]
> But what you consider E is tricky. In wetter climates and some riverscapes it is likely reasonably to assume that the range for your E is up to 100%. But you at least know its not more then that. If you can assume it to be 100%, or just live with this and side step it as your "historic expectation" but just treat it as a "maximum expectation", you now have a way out for this one.

> The maximum "active" floodplain extent is obviously bounded by that [valley bottom]. The problem, as the LCT folks will be quick to point out, is that in really arid areas with smaller hydrologic engines naturally, you would not have historically expected the entire valley bottom to be active, but it all would have been accessible. It is a start.

Not sure how practical (or even worthwhile) this is but is there a way to leverage the streamstats integration and use gages combined with the discharge estimates from streamstats to try and approximate some sort of discharge-stage relationship? Could be a QRIS feature that might help at least on a reach level to approximate what of the low lying areas *might* flood. You could use various flood intervals and the regional flow equations and that might help overcome the arid climates not getting quite as much water while wetter climates get more. Potentially using Jordan's cool floodplain accessibility raster (maybe a modified version that ignores human infrastructure for the sake of "historic condition") to help build an idea of potentially floodable areas. 

This is also restricted by how historic the gage data is (likely not historic enough) and limited by areas which have gages. Also wouldn't have a historic channel to accurately represent at which place in the valley bottom the water will be coming from. This might be a start to understanding an "expected" floodplain on a reach level though. 

Sidebar: the figure about the Production Grade Models Waterfall, the linebreak in the legend between Hybrid Channel/Riverscape Network Models and Non-Network Models doesn't read very well and looks more like Hybrid Channel/Riverscape and Network Models Non-Network Models. Also Riverscape Network Models is cut off. Not sure if this is something you made for this discussion/WIP or if this is a figure that needs to be spot free for presentations but thought I'd bring it up just in case.

Also pinging @emilySarge and @MB-Turnage as they might have some insights to this as well.

**Reply by joewheaton** [reply]
Good catch on the sidebar... font change... I will fix that.  I appreciate the attention to detail.

So QRiS actually has a feature that runs StreamStats for you already. Running hydraulic models is a step beyond what should be done in QRiS, but it is something some people will be tempted to do. So maybe considering what sort of  features to support and accomodate people doing that elsewhere in the future is good. 

Also, this is only something that makes sense to do IFF you have good bathymetery and high resolution topography of your riverscape. That does happen. I think for CJ's question though, what you said about "historic" is the problem. We don't tend to have "historic" regional curves that ago back far enough.  Even if we did, we only have contemporary topography to route floods across. 

But I like how you are thinking creatively...

**Comment by ral-chokhachy** [comment]
Great link here, and we are working on very similar concepts in the Great Basin with Wally Macfarlane and Joe's Lab--merging different products to get at riverscape health. Convening for some discussions would be fruitful.

I like the idea of O/E @chris3jordan  but moving from a single measure of E to a distribution of Es is probably more realistic/useful.  Then it would be good to contrast the distribution of Os relative to the distributions of Es in any given geo- and climatic setting. In the Great Basin, we don't have a lot of reference. Even in wilderness areas in this region (I know there are likely legacy effects, but if we ignore), streams are not ~100% connected to the valley bottoms. More a longitudinal and horizontal mosaic. This is what gets me to think of using distributions.

**Reply by chris3jordan** [reply]
@ral-chokhachy - yes, we do not know a singular <expected> or even <observed> value for any of these potential components of a riverscape impairment/integrity index (RII?), but I'm not sure that a distribution approach would be that practical. However, we certainly could think of the confidence or support for a best estimate of each metric (numerator and denominator).

Also, I'm thinking that we are not trying to estimate an <expected> state that involves spatial precision (where is what kind of channel reach), more like lateral extent of seasonal mesic veg, dominant veg type (driver of potential channel morpho state), some way to capture larger scope dynamics, 

I agree that we are in some way hampered by a pervasive lack of "reference" reaches, but that also can be of value, as we need to shake lose of the "make it look like over there" mentality. Reconstruct more from first principles than by guessing that reach A is (1) intact, and (2) what reach B is supposed to look like.

**Reply by ral-chokhachy** [reply]
I see your points @chris3jordan. I was thinking (as it relates to your last paragraph) importantly what is "intact" can be range--no presciptive set value.  I.e., a range of values may be acceptable. Using existing data to understand what this range of "what reach B is supposed to look like" may be reasonable.

**Comment by wally-mac** [comment]
@chris3jordan - I like your idea of a riverscape condition metric in the form of observed/expected. If we assume that an intact riverscapes is **E** and that they tend to have high lateral and vertical connectivity and more diverse flow paths often forced by structural instream elements (large wood and beaver dams). And we assume that degraded riverscapes often have low lateral and vertical connectivity contain a single simplified channel that rapidly transports water and sediment through the system, and are therefore only capable of supporting a small fraction of the historical riparian zone. Then, number of channels, channel length per unit valley length, late summer NDVI, proportion riparian vegetation, proportion of active floodplain, etc. as well as numbers of jams and jams are important indicators for this riverscape condition metric.

I think we can determine **E** by observing existing intact riverscape conditions (e.g. of areas where beavers are currently active or where there is a lot of large wood loading). I don’t think we have to rely on historical condition to get at **E**.
